blathering-barnacles/Bangazon-2
# Sell product

- User can click on the sell product button on the nav bar, the item will be added to the database
- User will see a form to fill out to add a product to the database

To Test
- Navigate to localhost:8000/sell
- fill out the new product form
- click submit and check the database to see the item

!!!User and customer must have the same id!!!

items do no appear in the app currently
